packages/formstr-app/src/containers/CreateForm/index.tsx

```diff
@@ -1,14 +1,15 @@
 import Sidebar from "./components/Sidebar";
 import { QuestionsList } from "./components/QuestionsList";
 import FormBuilderProvider from "./providers/FormBuilder";
+import Settings from "./components/Settings";
 
 function FormBuilder() {
   return (
     <FormBuilderProvider>
       <div style={{ display: "flex", maxWidth: "100vw" }}>
         <Sidebar />
         <QuestionsList />
-        <div style={{ minWidth: "20%" }}>right sidebar</div>
+        <Settings />
```

Comment:
if we are calling the right Sidebar as Settings, does it make sense to rename the left one to something else? InputBar maybe? open to better names.

Reply:
let's keep this sidebar as of now, as in future, we might add more sections in the sidebar other than input block